source/glbinding/source/RingBuffer.hpp

```diff
@@ -0,0 +1,243 @@
+#pragma once
+
+#include <iostream>
```

Comment:
It seems like you removed all debug output so this (and also the ```iomanip``` headers are superfluous?